targets/targets.json

```diff
@@ -8053,29 +8053,33 @@
         "inherits": ["Target"],
         "core": "Cortex-M4F",
         "public": true,
-        "extra_labels": ["RDA", "UNO_91H", "FLASH_CMSIS_ALGO"],
+        "extra_labels": ["RDA", "UNO_91H", "FLASH_CMSIS_ALGO", "RDA_EMAC"],
         "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
         "macros": ["TWO_RAM_REGIONS", "CMSIS_NVIC_VIRTUAL", "CMSIS_NVIC_VIRTUAL_HEADER_FILE=\"RDA5981_nvic_virtual.h\""],
         "device_has": [
-            "USTICKER",
-            "PORTIN",
-            "PORTOUT",
-            "PORTINOUT",
-            "INTERRUPTIN",
-            "SERIAL",
-            "STDIO_MESSAGES",
-            "PWMOUT",
-            "SPI",
-            "SLEEP",
-            "ANALOGIN",
-            "FLASH",
-            "TRNG"
+        "USTICKER",
```

Comment:
I've noticed this should be aligned as it was previously (4 spaces more to the right) ?